### ISSUE TITLE
How to specify security schemes for API Endpoints

### ISSUE THREAD

**Issue by kushal-ti** [TestBoxLab/chalice-spec#16]
Hi, first of all, i just want to say this is in an incredibly useful tool and has made my life much easier. 

I was wondering if there's a way to specify the required security schema for an API route, such as bearer Auth for example yet. For instance, with the apispec flask plugin, you can specify security schema in the docstring of the method as seen here https://apispec.readthedocs.io/en/latest/#example-application. 

I have added bearer auth to my swagger definition itself, using the following code 

```python
bearer_auth = {"type": "http", "scheme": "bearer"}
spec.components.security_scheme("Auth", bearer_auth)
```

which has resulted in my rendered API Swagger looking as such

<img width="1249" alt="Screenshot 2023-05-16 at 6 50 05 PM" src="https://github.com/TestBoxLab/chalice-spec/assets/88190553/6439c49d-31df-4b32-8746-a374ca8f1567">

<img width="1106" alt="Screenshot 2023-05-16 at 6 51 35 PM" src="https://github.com/TestBoxLab/chalice-spec/assets/88190553/b62b66b8-e59c-4a72-acbe-5562ea40299f">

But i can't figure out how to add the bearer auth as a security to specific routes 

I've been through the source code and far as i can tell there isn't an inbuilt way to do this already, but just wanted to confirm. If it is not, i'd like to contribute the same if you're accepting PRs

**Comment by kushal-ti** [comment]
Update, i looked into it and it was a very basic edit, just 2 lines of code changes, so i made them anyway, have raised a pull request here https://github.com/TestBoxLab/chalice-spec/pull/17

Please do review and merge if acceptable

**Comment by kushal-ti** [comment]
@jakemwood noticed the test failures, will edit the tests in accordance with the new changes

**Comment by jakemwood** [comment]
Hi @kushal-ti ! Sorry for the delay here. This has been released in v0.7.0!
